src/jit/importer.cpp

```diff
@@ -6462,6 +6463,15 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         call->gtCall.callSig  = new (this, CMK_CorSig) CORINFO_SIG_INFO;
         *call->gtCall.callSig = calliSig;
 #endif // DEBUG
+
+        if (IsTargetAbi(CORINFO_CORERT_ABI))
+        {
+            bool managedCall = (calliSig.callConv & GTF_CALL_UNMANAGED) == 0;
+            if (managedCall)
+            {
+                call->gtFlags |= GTF_CALL_M_FAT_POINTER_CHECK;
```

Comment:
>call->gtFlags |= GTF_CALL_M_FAT_POINTER_CHECK; [](start = 16, length = 46)

GTF_CALL_M_FAT_POINTER_CHECK - is to be set on call->gtCallMoreFlags not on gtFlags!

GTF_CALL_M prefix denotes that it is a flag to be set on gtCallMoreFlags.

Note that the value of this flag overlaps with GTF_SPILLED_OP2